src/plugins/administrative_templates/ui/presentationbuilder.cpp

```diff
@@ -107,6 +107,107 @@ QHBoxLayout *createCaptions()
     return horizontalLayout;
 }
 
+QMap<std::string, QString> loadListFromRegistry(AbstractRegistrySource &m_source, const std::string &key, const std::string &prefix)
+{
+    QMap<std::string, QString> items;
+    std::vector<std::string> valueNames = m_source.getValueNames(key);
+    
+    // finding and erase **delvals.
+    for (auto it = valueNames.begin(); it != valueNames.end(); ++it) {
+        if (QString::fromStdString(*it).compare("**delvals.", Qt::CaseInsensitive) == 0) {
+            it = valueNames.erase(it);
+
+            // std::vecotor<T>::erase() returning iterator that pointing 
+            // to the next element after erased
+            --it; 
+            break;
+        }
+    }
+    
+    for (auto &valueName : valueNames) {
+        items[valueName] = 
+                m_source.getValue(key, valueName).value<QString>();
+    }
+    return items;
+}
+
+void cleanUpListInRegistry(AbstractRegistrySource &m_source, const std::string &key)
```

Comment:
In variable name prefix `m_` indicates member variable of a class. In this case we do not need this prefix because it is a function parameter.